### PR TITLE
Add GigaFren Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/giga_fren/giga_fren.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+_DESCRIPTION = """\
+Giga-word corpus for French-English from WMT2010 collected by Chris Callison-Burch
+2 languages, total number of files: 452
+total number of tokens: 1.43G
+total number of sentence fragments: 47.55M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/giga-fren.php"
+_CITATION = """\
+J. Tiedemann, 2012, Parallel Data, Tools and Interfaces in OPUS. In Proceedings of the 8th International Conference on Language Resources and Evaluation (LREC 2012)
```
Comment (lhoestq) [inline datasets/giga_fren/giga_fren.py:30]: can you use the bibtex format here please ?

File: datasets/giga_fren/giga_fren.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                x = x.strip()
+                y = y.strip()
+                result = (
+                    sentence_counter,
+                    {
+                        "id": str(sentence_counter),
+                        "translation": {l1: x, l2: y},
+                    },
+                )
+                sentence_counter += 1
```
Comment (lhoestq) [inline datasets/giga_fren/giga_fren.py:100]:
```suggestion
```

**Review by lhoestq — COMMENTED** [review]
Nice thanks !

Left a few comments

**Comment by abhishekkrthakur** [comment]
@lhoestq fixed

File: datasets/giga_fren/giga_fren.py [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+_DESCRIPTION = """\
+Giga-word corpus for French-English from WMT2010 collected by Chris Callison-Burch
+2 languages, total number of files: 452
+total number of tokens: 1.43G
+total number of sentence fragments: 47.55M
+"""
+_HOMEPAGE_URL = "http://opus.nlpl.eu/giga-fren.php"
+_CITATION = """\
+@InProceedings{TIEDEMANN12.463,
+  author = {J�rg Tiedemann},
```
Comment (lhoestq) [inline datasets/giga_fren/giga_fren.py:31]:
```suggestion
  author = {J{\"o}rg Tiedemann},
```

**Review by lhoestq — APPROVED** [review]
Thanks :)

**Pull request merged**